task07-consume-real-data/json-data.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary js with json related functions
+ */
+
+ function sortArray(array){
+    array.sort((a,b) => {
+        let a1 = a.name.toLowerCase(),
+            b1 = b.name.toLowerCase();
+    
+        if (a1 < b1) {
+            return -1;
+        }
+        if (a1 > b1) {
+            return 1;
+        }
+        return 0;
+    });
+}
+
+export async function fetchOnePieceData(){
+    const OnePieceUrl = 'https://static.akurey.com/trainings/OnePieceInformation.json';
+    
+    try {
+        let onePieceInfo = await fetch(OnePieceUrl)
+        .then( (response) => response.json());
```

Comment:
You can do something like:

try{
const onePieceData = await fetch(url);
const response = await onePieceData.json();
return response;
}

In this case you don't use the promises methods, just the await/async function :)